swan-lake/integration-technology-tools/health-tool.md

```diff
@@ -3,7 +3,198 @@ layout: ballerina-health-support-left-nav-pages-swanlake
 title: Health tool (FHIR/HL7)
 description: The health tool generates accelerators for developing healthcare integrations in Ballerina.
 keywords: ballerina, programming language, health, contract
-permalink: /learn/edi-tool/
+permalink: /learn/health-tool/
 active: health-tool
 intro: The health tool generates accelerators for developing healthcare integrations in Ballerina.
 ---
+
+The Health tool can generate accelerators primarily focussed on the [Fast Healthcare Interoperability Resources (FHIR) standard](https://www.hl7.org/fhir/overview.html). The Health tool is supported from Ballerina version 2201.7.0 (Swan Lake Update 7) onwards.
```

Comment:
remove the `standard` word from link text. i.e. this should be `[Fast Healthcare Interoperability Resources (FHIR)](https://www.hl7.org/fhir/overview.html) standard`

Reply:
Fixed.